items/items-perm.md

```diff
@@ -79,15 +87,29 @@ Whether (true/false) the group is the owner of this item. Implies the maximum le
 
 ## Aggregation of permissions from multiple sources
 
-We call *aggregation* the merge of the multiple granted permissions (from possibily different groups) to one permission tuple (indexed by group, item).
+We call *aggregation* the merge of multiple granted permissions to one permission tuple `(group, item)` representing the permissions one group has on an item.
+Those granted permissions may have initially been given to different groups (the group itself or any of its ancestors).
```

Comment:
Confusing. This may describe propagation as well.

Reply:
fixed, also added how this notion is translated for users